manticore/core/smtlib/visitors.py

```diff
@@ -147,6 +147,7 @@ def visit_Operation(self, expression, *operands):
         else:
             self._print('...')
         self.indent -= 2
+        return ''
```

Comment:
Does making _method stop iterating when there is a visit_operation defined change something?  (insted when the visit_operation is defined _and_ returns not None)